traitcuration/traits/datasources/dummy.py

```diff
@@ -0,0 +1,148 @@
+"""
+This module holds dummy data to import during development
+"""
+from ..models import OntologyTerm, MappingSuggestion, Trait, Mapping, User, Review
+
+
+def import_dummy_data():
+    Review.objects.all().delete()
+    MappingSuggestion.objects.all().delete()
+    Mapping.objects.all().delete()
+    Trait.objects.all().delete()
+    OntologyTerm.objects.all().delete()
+    User.objects.all().delete()
+    # ONTOLOGY TERMS
+    term_current1 = OntologyTerm(label='Diabetes mellitus', curie='EFO:0000400',
+                                 iri='http://www.ebi.ac.uk/efo/EFO_0000400', status='current')
+    term_current1.save()
+    term_current2 = OntologyTerm(label='digestive system disease', curie='EFO:0000405',
+                                 iri='http://www.ebi.ac.uk/efo/EFO_0000405', status='current')
+    term_current2.save()
+    term_ni1 = OntologyTerm(label='Insulin-resistant diabetes mellitus', curie='HP:0000831',
+                            iri='http://purl.obolibrary.org/obo/HP_0000831', status='needs_import')
+    term_ni1.save()
+    term_ni2 = OntologyTerm(label='breast-ovarian cancer, familial, susceptibility to, 3', curie='MONDO:0013253',
+                            iri='http://purl.obolibrary.org/obo/MONDO_0013253', status='needs_import')
+    term_ni2.save()
+    term_ai2 = OntologyTerm(label='pancreatic cancer, susceptibility to, 4', curie='MONDO:0013685',
+                            iri='http://purl.obolibrary.org/obo/MONDO_0013685', status='awaiting_import')
+    term_ai2.save()
+    term_nc1 = OntologyTerm(label='Hypogonadism, diabetes mellitus, alopecia, mental retardation and \
+          electrocardiographic abnormalities', description='The description for Hypogonadism, diabetes \
+          mellitus, alopecia, mental retardation and electrocardiographic abnormalities ',
+                            cross_refs='MONDO:0013685,HP:0000400', status='awaiting_creation')
+    term_nc1.save()
+    term_nc2 = OntologyTerm(label='Familial cancer of breast, 2',
+                            description='Description for familial cancer of breast, 2',
+                            cross_refs="Orphanet:0000405", status='awaiting_creation')
+    term_nc2.save()
+    term_ac1 = OntologyTerm(label='Diastrophic dysplasia',
+                            description='Description for Diastrophic dysplasia',
+                            cross_refs="", status='awaiting_creation')
+    term_ac1.save()
+    term_obsolete = OntologyTerm(label='Diastrophic dysplasia', curie='Orphanet:628',
+                                 iri='http://www.orpha.net/ORDO/Orphanet_628', status='obsolete')
+    term_obsolete.save()
+    term_deleted = OntologyTerm(label='Spastic paraplegia', curie='HP:0001258',
+                                iri=' http://purl.obolibrary.org/obo/HP_0001258', status='deleted')
```

Comment:
Is the space at the beginning of IRI intentional to make it invalid? I think this might be problematic. For example, `requests` library for Python apparently strips leading/trailing spaces from addresses. I suspect _some_ other services _might_ do the same, hence the behaviour might not be consistent.

I suggest another way of making the term appear "deleted": simply using an impossibly large HP ID but an otherwise correct IRI. This way, when we implement OLS updates further down the road, this dummy trait will never lose its deleted status:

```suggestion
    # CURIE and IRI are intentionally incorrect so that the term stays deleted after OLS updates
    term_deleted = OntologyTerm(label='Spastic paraplegia', curie='HP:999999999',
                                iri='http://purl.obolibrary.org/obo/HP_999999999', status='deleted')
```